modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -76,23 +75,23 @@ export default function UpperToolbar({displayNames, languageIds, version}) {
 		}
 	};
 
-	const getXMLContent = () => {
+	const getXMLContent = (exporting) => {
 		let xmlContent;
 
-		if (currentEditor) {
+		if (currentEditor && !exporting) {
 			xmlContent = currentEditor.getData();
 		}
 		else {
 			xmlContent = serializeDefinition(
 				xmlNamespace,
 				{
 					description: definitionDescription,
-					name: definitionName,
+					name: definitionTitle,
```

Comment:
@eliasantosf In this case we need to use definitionName instead of definitionTitle.